humans/hulcher_byron/details.json

```diff
@@ -1,14 +1,14 @@
 {
-  "title": "",
+  "title": "Web Developer",
   "slug": "byron-hulcher",
   "role": [
     "Speaker"
   ],
   "twitter": "hypirlink",
   "facebook": "",
   "linkedin": "",
-  "personalWebsite": "http:\/\/hypir.link",
-  "companyWebsite": "",
+  "personalWebsite": "http://hypir.link",
+  "companyWebsite": "http://nymag.com",
```

Comment:
We need the escapes for the slashes and then I'll merge this for you. Thanks @byronhulcher!